src/test/java/com/google/firebase/messaging/MulticastMessageTest.java

```diff
@@ -47,6 +48,7 @@ public void testMulticastMessage() {
         .setApnsConfig(APNS)
         .setWebpushConfig(WEBPUSH)
         .setNotification(NOTIFICATION)
+        .setFcmOptions(FCM_OPTIONS)
```

Comment:
Update the assertMessage helper method to verify that the FcmOptions has been set.